en/services/component_information.md

```diff
@@ -179,18 +180,78 @@ Specifically, the following information is provided:
 
 A GCS can provide a UI for testing outputs based on the configured output functions, by iterating over all output channels and collecting the configured actuator output functions, and then utilizing the `MAV_CMD_ACTUATOR_TEST` command.
 
+## Translation
 
-## Open Issues
+At high-level, metadata translation works as follows:
+
+- The metadata provider sets the `translationUri` in [general metadata file](#COMP_METADATA_TYPE_GENERAL) for each metadata type that supports translation.
+  Note that the URL has no associated CRC as the translation data can change independently of metadata (for example, adding or changing translations).
+- The `translationUri` URL points to a summary JSON file that contains links to the separate files that contain each translation of the particular metadata type.
+  The translation summary JSON file also contains modification timestamps for each linked translation file so that a GCS can determine whether a particular file has been updated.
+  The translation files are a (compressed) file in [TS file format](https://doc.qt.io/qt-6/linguist-ts-file-format.html).
```

Comment:
@bkueng Is this correct?
(i.e. compression format). Not so concerned about this as it is Internet to GCS.
```suggestion
  The translation files are in [TS file format](https://doc.qt.io/qt-6/linguist-ts-file-format.html) and may optionally, also be compressed in .bz format.
```

Reply:
`.xz`
Yes it is less relevant, but still, it might be a verbose xml file.

Reply:
Thanks. But should this be optional or mandatory? If optional then text above good, if not optional:

```suggestion
  The translation files are .bz compressed in [TS file format](https://doc.qt.io/qt-6/linguist-ts-file-format.html).
```

Reply:
We can make it optional, it's easy to handle both in the receiver. And it should be `xz` not `bz`